src/Counter.elm

```diff
@@ -0,0 +1,63 @@
+module Counter exposing (update)
```

Comment:
You should usually expose `Model`, `Msg`, `init`, `view` (and possibly `subscriptions`) as well.